products/spanner/helpers/instance_config_helpers.py.erb

```diff
@@ -0,0 +1,8 @@
+def encode_request(request, module):
+    request['name'] =
+      "projects/%(project)/instanceConfigs/%(name)" % module.params
```

Comment:
This seems to be the only format string using the % type of formatting, maybe change it for consistency?

Reply:
Done.